pkg/scheduler/framework/plugins/podtopologyspread/filtering_test.go

```diff
@@ -1400,6 +1400,32 @@ func TestSingleConstraint(t *testing.T) {
 				"node-b": framework.Unschedulable,
 			},
 		},
+		{
+			// In this unit test, we don't run NodeAffinity Predicate, so node-b
+			// still expected to be fits;
```

Comment:
In a unit test, NodeAffinity plugin is not involved, so node-b still fits.

Reply:
done.